index.js

```diff
@@ -94,20 +95,20 @@ app.use((req, res, next) => {
 
 // middleware function to check for logged-in users
 var sessionChecker = (req, res, next) => {
-    if (req.session.user && req.cookies.user_sid) {
-        res.redirect('/dashboard');
+    if (!req.session.user && !req.cookies.user_sid) {
+        res.redirect('/login')
     }
     else {
-        next();
+        next()
```

Comment:
having this the other way around said too many redirects when I tested it, so did a negative check instead

Reply:
Thanks for these explanatory notes :smile: